extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmCdiProcessor.java

```diff
@@ -191,13 +191,10 @@ void generateJpaConfigBean(HibernateOrmRecorder recorder,
     // These beans must be initialized at runtime because their initialization
     // depends on runtime configuration (to activate/deactivate a persistence unit)
     @Record(ExecutionTime.RUNTIME_INIT)
+    @Consume(JdbcDataSourceBuildItem.class) // just make sure the datasources are initialized
     @BuildStep
-    void generateDataSourceBeans(HibernateOrmRecorder recorder,
+    void generateHibernateBeans(HibernateOrmRecorder recorder,
             List<PersistenceUnitDescriptorBuildItem> persistenceUnitDescriptors,
-            ImpliedBlockingPersistenceUnitTypeBuildItem impliedBlockingPersistenceUnitType,
```

Comment:
do we need to put this one as consumed as well ?

Reply:
I do not think so, it was previously used in this method and was probably forgotten about when refactoring, but I don't think it's used for ordering build steps.